tests/Fixtures/TestBundle/Entity/Foo.php

```diff
@@ -24,7 +23,7 @@
  * @author Vincent Chalamon <vincentchalamon@gmail.com>
  *
  * @ApiResource(attributes={
- *     "order"={"name", "DESC"}
+ *     "order"={"bar": "ASC", "name": "DESC"}
```

Comment:
Just some nitpicking 😬: it would be great to also allow `"order"={"bar", "name"}` (that would be equivalent to ` "order"={"bar": "ASC", "name": "ASC"}`. It's just some syntactic sugar but it improves the overall DX.